shell/browser/native_window_views.cc

```diff
@@ -157,6 +162,10 @@ NativeWindowViews::NativeWindowViews(const gin_helper::Dictionary& options,
                                      NativeWindow* parent)
     : NativeWindow(options, parent),
       root_view_(std::make_unique<RootView>(this)),
+      // It's hacky to use GetPlatformNameForTest to figure out if we run under
+      // X11 but that's the best worst way at the moment.
+      is_x11_(ui::OzonePlatform::GetInstance()->GetPlatformNameForTest() ==
+              "x11"),
```

Comment:
Should be wrapped in `#if defined(USE_OZONE)`.

Reply:
Oops, fixed!